pytket/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ pybind11_add_module(circuit
     binders/circuit/unitid.cpp
     binders/circuit/boxes.cpp
     binders/circuit/classical.cpp
+    binders/circuit/pool.cpp
```

Comment:
This is personal taste: I find the word pool a bit non-descriptive. Would you consider `decompositions` or `circuit.library` for example?

Reply:
changed to library